components/admin_console/team_channel_settings/team/details/team_modes.tsx

```diff
@@ -28,7 +37,7 @@ const SyncGroupsToggle = ({syncChecked, allAllowedChecked, allowedDomainsChecked
                 id='admin.team_settings.team_details.syncGroupMembersDescr'
                 defaultMessage='When enabled, adding and removing users from groups will add or remove them from this team. The only way of inviting members to this team is by adding the groups they belong to. <link>Learn More</link>'
                 values={{
-                    link: (msg) => (
+                    link: (msg: string) => (
                         <a
                             href='https://www.mattermost.com/pl/default-ldap-group-constrained-team-channel.html'
                             referrer='noreferrer'
```

Comment:
Type check is failing because of this: Property 'referrer' does not exist on type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'. I wanted to cross check before deleting it.